packages/farcaster-js-neynar/src/NeynarAPIClient.ts

```diff
@@ -0,0 +1,38 @@
+import { NeynarV1APIClient } from './v1/';
+import { NeynarV2APIClient } from './v2/';
+import { AxiosInstance } from 'axios';
+import { silentLogger, Logger } from './logger';
+
+export class NeynarAPIClient {
+  private readonly logger: Logger;
+
+  public readonly clients: {
+    v1: NeynarV1APIClient
+    v2: NeynarV2APIClient
+  };
```

Comment:
Nice. Yeah this is cool. WDYT about `NeynarAPIClient.v1` instead of `NeynarAPIClient.clients.v1`?

Reply:
good idea